### PR TITLE
Markdig migration

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

**Comment by shirhatti** [comment]
@mairaw Is this PR still relevant? Or was it superseded by your other PRs?

**Comment by mairaw** [comment]
Yes, still relevant @shirhatti. These are the automated changes made by the markdig migration script.

File: iis/application-frameworks/install-and-configure-php-applications-on-iis/install-joomla-on-iis.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ In order for Joomla to work correctly with IIS 7 and above, it is necessary to i
 
 First, download the latest stable release of [Joomla](http://www.joomla.org/). For this article, Joomla version 1.5.14 was used (future versions are likely to be similar to install). Uncompress the Joomla files and copy them to **C:\Inetpub\wwwroot\joomla** (or another directory of your choosing).
 
-Before running the Joomla installer, create the configuration file **Configuration.php****,** which Joomla uses to store key configuration settings. Assign write permissions to it with the command (from the installation directory):
+Before running the Joomla installer, create the configuration file **Configuration.php**<strong>,</strong> which Joomla uses to store key configuration settings. Assign write permissions to it with the command (from the installation directory):
```
Comment (shirhatti) [inline iis/application-frameworks/install-and-configure-php-applications-on-iis/install-joomla-on-iis.md]: Seems weird?

Comment (mairaw) [reply iis/application-frameworks/install-and-configure-php-applications-on-iis/install-joomla-on-iis.md]: the original source as problematic. I'll fix that.

File: iis/develop/extending-iis-configuration/configuration-extensibility.md [hunk 2]
```diff
@@ -56,11 +56,53 @@ The Windows group IIS\_IUSRS must have write permissions to the directory so tha
 
 Although Step 1 is complete in terms of our example, it is appropriate to discuss the schema files. In the schema above, we simply created a new configuration section **simpleLogging** that exists under **system.webServer** and specified a custom attribute. However, you can easily create more complex custom configuration with collections, elements and attributes. The table below shows some examples, but the best way to learn is to look at the schema file for the IIS configuration. Find it at `%windir%\system32\inetsrv\config\schema\IIS\_schema.xml`.
 
-| Type | Schema Info and Examples |
-| --- | --- |
-| **attribute** | [!code-console[Main](configuration-extensibility/samples/sample4.cmd)] <br> **Example:** <br> [!code-xml[Main](configuration-extensibility/samples/sample5.xml)] |
-| **element** | [!code-xml[Main](configuration-extensibility/samples/sample6.xml)] <br> **Example:** <br> [!code-xml[Main](configuration-extensibility/samples/sample7.xml)] |
-| **collection** | [!code-xml[Main](configuration-extensibility/samples/sample8.xml)] <br> **Example:** <br> [!code-xml[Main](configuration-extensibility/samples/sample9.xml)] |
+:::row:::
```
Comment (shirhatti) [inline iis/develop/extending-iis-configuration/configuration-extensibility.md]: Is this a valid markdown table? Never seen this syntax before

File: iis/docfx.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "build": {
-    "markdownEngineName": "dfm",
+    "markdownEngineName": "markdig",
```
Comment (shirhatti) [inline iis/docfx.json]: Can this be changed in this PR? Or should we wait till all markdig migration PRs are merged?

**Pull request merged**